app/assets/stylesheets/layouts/_basics.scss

```diff
@@ -1,4 +1,5 @@
 h2 {
+  margin: 23px 0px 23px 20px;
   font-size: 25px;
```

Comment:
Font-Size should not be picked randomly. Let's have a H1 Size, a H2 Size and a p-size :), or maybe 2-3 smaller font-sizes like 18-14-10px

Reply:
agreed but here i am picking all the h2, h3 and p. Isn't it correct?